ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -232,6 +233,7 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
 
         tokenSet.registerOnUpdate(for: self) { [weak self] in
             self?.updateTitleAndBackgroundColors()
+            self?.updateLeadingViewColor()
             self?.updateAccessoryButtonTitleColor()
```

Comment:
Could you move all these updates to just one `updateTokenizedValues()` or something function?